compose-macros/src/lib.rs

```diff
@@ -93,7 +93,8 @@ macro_rules! compose_extrinsic_with_nonce {
 
             debug!("Composing generic extrinsic for module {:?} and call {:?}", $module, $call);
 
-            let call = $crate::compose_call!($api.metadata().clone(), $module, $call $(, ($args)) *);
+			let metadata = $api.metadata().clone();
```

Comment:
Do we need this clone?

Reply:
Good Input. Doesn't look like it does, removed it :)